test/ui/test/button-table-edit.jsx

```diff
@@ -13,6 +13,34 @@ describe('ButtonTableEdit', function() {
 	beforeEach(Utils.createAlloyEditor);
 	afterEach(Utils.destroyAlloyEditor);
 
+	it('should create a 1x1 table by default when user selects a negative value for rows or columns', function() {
```

Comment:
As per [`liferay/no-it-should` ESLint rule](https://github.com/liferay/eslint-config-liferay/blob/HEAD/plugins/eslint-plugin-liferay/docs/rules/no-it-should.md), I'd suggest changing this from "should create" to "creates".

Reply:
updated whole file